systemd/coreos-installer-pre.target

```diff
@@ -0,0 +1,4 @@
+[Unit]
+Description=Prepare for CoreOS Installer Target
+
+Requires=coreos-installer-devices.service
```

Comment:
Bikeshed: `coreos-installer-disable-auto-activation.service`? For `oneshot` services, I like names that have a verb in them to make it really clear what it does.